values/kubeflow-pipelines/kubeflow-pipelines.gotmpl

```diff
@@ -50,6 +50,12 @@ mlPipelineVisualizationserver:
 
 mysql:
   resources: {{- $kfp.resources.mysql | toYaml | nindent 4 }}
+  storage:
+    enabled: {{ $kfp | get "mysql.storage.enabled" true }}
```

Comment:
the `"mysql.storage.enabled"` does not exists in the values-schema so it should not be used here.